packages/11ty/_plugins/transforms/outputs/pdf/transform.js

```diff
@@ -12,9 +15,12 @@ const { JSDOM } = jsdom
  * @param      {String}  content      Output content
  * @return     {Array}   The transformed content string
  */
-module.exports = function(eleventyConfig, collections, content) {
+module.exports = async function(eleventyConfig, collections, content) {
   const pageTitle = eleventyConfig.getFilter('pageTitle')
   const slugify = eleventyConfig.getFilter('slugify')
+  const citation = eleventyConfig.getFilter('citation')
+  const quirePDFConfig = eleventyConfig.globalData.config.pdf
```

Comment:
It may be unnecessary to specify `quire*` here in the variable names?
```suggestion
  const pdfConfig = eleventyConfig.globalData.config.pdf
```

Reply:
Implemented this in an earlier sprint.